### ISSUE TITLE
Acc on Cifar 100 with bsz 256

### ISSUE THREAD

**Issue by JongMokKim** [HobbitLong/SupContrast#8]
First of all, thank you for the great work! your repo is really helpful for me

I have been training the network with your code, but the performance is much higher than your report. Could you check it is normal? 

At pretraining stage, my experimental setting is same as readme EXCEPT for bsz (1024 --> 256 due to the lack of GPU)
and linear evaluation stage, it is extremely equal as your guide. 

but my accuracy is almost 87% (your report is 76.5)

below is my tensorboard graphs

![image](https://user-images.githubusercontent.com/40392910/84107281-2e5e5700-aa58-11ea-994f-9e88ce4aa6d6.png)

Could you give your opinion?

Thank you !

**Comment by JongMokKim** [comment]
I made a mistake, so close this issue!

**Comment by seekingup** [comment]
Hi, I met the exactly same problem with you on cifar100. My batch size is 256. How did you solve the problem? Could you tell me what's the problem is? Thank you?
